### ISSUE TITLE
AlphaFold2_multimer2 

### ISSUE THREAD

**Issue by JohnJPaul** [YoshitakaMo/localcolabfold#73]
Will the local collar fold be updated to include the AlphaFold2_multimer2 ?

**Comment by YoshitakaMo** [comment]
Yes, you can use AlphaFold_multimer_v2. Type `./update_linux.sh .` to update colabfold to the latest version. After the update, the new weight v2 parameters will be automatically downloaded and used for the prediction when you started multimer prediction.

**Comment by JohnJPaul** [comment]
Thank you I am closing the issue
